chalice/package.py

```diff
@@ -241,6 +241,17 @@ def _inject_restapi_outputs(self, template):
     # noop methods to indicate we've accounted for these
     # resources.
 
+    def _generate_websocketapi(self, resource, template):
+        # type: (models.WebsocketAPI, Dict[str, Any]) -> None
+        message = (
+            "Unable to package chalice apps that use any websocket "
+            "decorators. @app.on_ws_connect, @app.on_ws_disconnect, "
+            "@app.on_ws_message. CloudFormation does not support "
```

Comment:
As of Feb 8, this is no longer true - CloudFormation has support for API Gateway v2: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-api.html

Reply:
We should look into this.  My concern is that we're using SAM to generate the event handlers to integrate with lambda, and it doesn't look like SAM supports web sockets yet https://github.com/awslabs/serverless-application-model/issues/727.

I'd be interested to see what a sample template looks like, but I'm a little concerned about going down this route especially if part of a resource comes from SAM and part comes from CFN.  It's not clear if we can migrate to the SAM version, when support is added, in a backwards compatible way.

Worth looking into, but given the size of this PR I'd suggest adding it as a follow up PR (assuming we find a reasonable way to do it).